worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -293,6 +293,9 @@ private[celeborn] class Worker(
 
   private val masterClient = new MasterClient(internalRpcEnvInUse, conf, true)
 
+  private val rackResolver = new CelebornRackResolver(conf)
+  private val networkLocation = rackResolver.resolve(host).getNetworkLocation
```

Comment:
Do we want to add to constructor? I think it is better to leave as is, since networkLocation is not part of the constructor currently. It is just a field: https://github.com/akpatnam25/incubator-celeborn/blob/CELEBORN-1313/common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala#L45